source/georef/georef.h

```diff
@@ -256,7 +257,8 @@ struct GeoRef {
     template <class Archive>
     void save(Archive& ar, const unsigned int) const {
         ar& ways& way_map& graph& offsets& fl_admin& fl_way& pl& projected_stop_points& admins& admin_map& pois& fl_poi&
-            poitypes& poitype_map& poi_map& synonyms& ghostwords& poi_proximity_list& nb_vertex_by_mode;
+            poitypes& poitype_map& poi_map& synonyms& ghostwords& poi_proximity_list& nb_vertex_by_mode&
+                inversed_nb_vertex_by_mode;
```

Comment:
because  

First surprisingly, the division is much slower the multiplication, the profiling showed that this division operation took 1-2% under my benchmark settings

Second I have to serialize it because there is no initialization after the data loading...

Reply:
@kinnou02 You're right!

Reply:
done